docs/server/source/production-deployment-template/architecture.rst

```diff
@@ -0,0 +1,69 @@
+Architecture of an IPDB Node
+============================
+
+An IPDB Production deployment is hosted on a Kubernetes cluster and includes:
+
+* NGINX, OpenResty, BigchainDB and MongoDB
+  `Kubernetes Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+* NGINX, OpenResty, BigchainDB, Monitoring Agent and Backup Agent
+  `Kubernetes Deployments <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_.
+* MongoDB `Kubernetes StatefulSet <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_.
+* External systems like `3scale <https://3scale.net>`_,
```

Comment:
How about 3rd party services/infrastrucutre? Also we can remove the `like`, since we only have these.

Reply:
We also use a certificate signing authorities and DNS providers. But these are the main ones.
Reword it to `Third party systems like...`?